buildSrc/src/main/java/Dependencies.kt

```diff
@@ -34,31 +34,32 @@ object Versions {
 
     const val mozilla_glean = "33.0.4"
 
-    const val material = "1.1.0"
+    const val material = "1.2.1"
     const val nearby = "17.0.0"
 
     object AndroidX {
         const val annotation = "1.1.0"
-        const val appcompat = "1.1.0"
+        const val appcompat = "1.2.0"
         const val browser = "1.2.0"
         const val cardview = "1.0.0"
-        const val constraintlayout = "1.1.3"
-        const val core = "1.2.0"
-        const val fragment = "1.1.0"
-        const val recyclerview = "1.0.0"
-        const val test = "1.2.0"
-        const val test_ext = "1.1.1"
-        const val espresso = "3.2.0"
+        const val constraintlayout = "2.0.3"
```

Comment:
They did a quick follow with 2.0.4 to fix an NPE and recommend using this version: https://developer.android.com/jetpack/androidx/releases/constraintlayout#2.0.4

Reply:
Thanks! Will update this.